.circleci/config.yml

```
@@ -237,7 +237,7 @@ jobs:
                   tag=$corresponding_milestone_title
                   change_notes=$(./gradlew getChangelog --no-header -q)
                   hub release create \
-                    --attach "$(./gradlew -q printPluginZipPath)" \
+                    --attach "$(./gradlew printSignedPluginZipPath)" \
```

```suggestion
                    --attach "$(./gradlew -q printSignedPluginZipPath)" \
```
To make sure no spam slips through